plugins/test-management-resources/src/components/test-case/TestCasesList.svelte

```diff
@@ -21,11 +21,11 @@
 
   import testManagement from '../../plugin'
 
-  export let baseQuery: DocumentQuery<Doc> = {}
+  export let query: DocumentQuery<Doc> = {}
   let testCases: number
 
-  const query = createQuery()
-  $: query.query(testManagement.class.TestCase, baseQuery, (res) => {
+  const docQuery = createQuery()
+  $: docQuery.query(testManagement.class.TestCase, query, (res) => {
     testCases = res.length
```

Comment:
Looks like we can use `total` here

```
docQuery.query(testManagement.class.TestCase, query, (res) => {
      testCases = res.total
    },
    { total: true, limit: 1 }
  )
```

Reply:
yep, it is a bad idea to load all testcases just to count them

Reply:
Thanks, fixed